### ISSUE TITLE
Bind to SD card only gives r/o access

### ISSUE THREAD

**Issue by RaXorX** [Magisk-Modules-Repo/com.piyushgarg.rclone#36]
I tried binding my mount points to SD card. One, it still mounted the point in /mnt/cloud and two, the one in /sdcard/cloud was only read access. Is this an issue or do I need to pass additional parameters to the param file. I had hoped that it would pick the default ones from the settings automatically, while using the additional ones I passed to Mount in SD card. 

Am I missing something here?

**Comment by ddogishere** [comment]
Hi can you help me understand please add I'm a total noob.
are you able to copy more than 1gb+ files to mnt/cloud? 
So I'm theory it will be using your remote (gdrive) storage and not local storage?

**Comment by RaXorX** [comment]
@ddogishere Yes, I was able to copy 1gb+ files to and forth, but it was quite slow. I'm not sure why, maybe it is related to multi-threading.
